publishing-bot/configs/example

```diff
@@ -0,0 +1,15 @@
+# the Docker repo for the bot image. Should be writable for you.
+DOCKER_REPO = <your-docker-user>/k8s-publishing-bot
```

Comment:
I think we can remove the example config from this repo and keep just the `kubernetes` and `kubernetes-nightly` ones.

We can keep the example ones in the publishing-bot repo to act as a reference for anyone who'd want to deploy their own publishing-bot instance.

Reply:
+1 from me